bin.js

```diff
@@ -37,11 +41,27 @@ const packageDetails = require(path.join(__dirname, "package.json"));
     let allowed = program.opts().allow;
     allowed = (allowed || "").split(",").filter((r) => r);
     let ignoreShas = program.opts().ignoreShas;
+    let only = program.opts().only;
+
+    let [onlyOwner, onlyRepo]  = [null,null];
+    let onlyVersion = null;
+    if ( only ){
+      let parts = only.split("/");
+      if ( parts.length != 2 ){
+        throw ( "Syntax for --only: account/repo or account/*");
+      }
+      [onlyOwner,onlyRepo] = parts;
+
+      parts = onlyRepo.split("@");
```

Comment:
Variable shadowing here. Could we use a more descriptive (and different) variable name?